src/source/examples/put-campaign-budgets-example3-v2.rst

```diff
@@ -0,0 +1,31 @@
+.. code-block:: javascript
+
+    PUT /api/sellside/campaign/124512/budgets
```

Comment:
should this not be a POST? Looks like you're creating an empty one here.

Reply:
there are no separate PUT/POST. There is only `PUT` for **create or replace**